Source/Example Measurements/NI-DCPower Source DC Voltage/NI-DCPower Measurement/NIDCPowerSourceDCVoltage.lvlib

```diff
@@ -0,0 +1,34 @@
+
+</Property>
+	<Item Name="Advanced" Type="Folder">
+		<Item Name="Build Assets" Type="Folder">
+			<Item Name="Post-Build Action.vi" Type="VI" URL="../Advanced/Build Assets/Post-Build Action.vi"/>
+		</Item>
+		<Item Name="MeasurementPlugin.lvclass" Type="LVClass" URL="../Advanced/MeasurementPlugin.lvclass"/>
+	</Item>
+	<Item Name="subVIs" Type="Folder">
+		<Item Name="Convert Location to Address String.vi" Type="VI" URL="../subVIs/Convert Location to Address String.vi"/>
+		<Item Name="DCPower Specific Logic.vi" Type="VI" URL="../subVIs/DCPower Specific Logic.vi"/>
+		<Item Name="Location to Grpc Device.vi" Type="VI" URL="../subVIs/Location to Grpc Device.vi"/>
```

Comment:
Add some form of NI-DCPower to the VI name and update it to use `niDCPower gRPC Device Server.ctl`

Reply:
Interesting. I didn't know that ctrl existed. Why don't the DCPower MeasurmentLink VIs use it? 

Do you still think this conversion VI should use the typedef ctrl even though the Init and Attach VIs don't use it? It will result in the a coercion dot, which isn't the end of the world.

Reply:
They should use it.
[AB#2287962](https://dev.azure.com/ni/DevCentral/_workitems/edit/2287962): NI-DCPower gRPC APIs shall use typedef control `niDCPower gRPC Device Server.ctl` in LabVIEW API for initialize and attach.

Reply:
I don't know if it can be changed now since NI-DCPower has shipped, but I don't see a reason for us not to use it.